app/locale/en_US/TIG_MyParcelBE.csv

```diff
@@ -158,6 +157,7 @@ TIG_MyParcelBE::Close,Close
 TIG_MyParcelBE::Choose a bpost pickup location,Choose a bpost pickup location
 "TIG_MyParcelBE::The selected shipment-type cannot be used, Pakjegemak-shipments can only be created with the normal shipment type.","The selected shipment-type cannot be used, 'Ship to bpost location' can only be created with the normal shipment type."
 TIG_MyParcelBE::MyParcel - Create Magento shipments (no labels),MyParcel - Create Magento shipments (no labels)
+TIG_MyParcelBE::Create MyParcel BE shipment
```

Comment:
Ik mis hier de vertaling. Dus met een ,

Reply:
Hier hoeft geen vertaling voor het is een `en_US` bestand

Reply:
Done!